### PR TITLE
Change design mobile and desktop, change delivery asset

### PR DESCRIPTION
**Pull request by CarlosOrdering**
ASANA task: https://app.asana.com/0/0/1199127996918363/f

change video: https://d.pr/v/2Iq4Iq

### CODE REVIEW

File: template/Pages/Home/styles.js [hunk 1]
```diff
@@ -92,6 +93,9 @@ export const StepsBlock = styled.div`
       text-align: center;
     }
   }
+  @media (max-width: 790px) {
```
Comment (quirozcarlos) [inline template/Pages/Home/styles.js:96]: @CarlosOrdering we're using mobile first in design, pls use `@media (min-width)`

File: template/Pages/Home/styles.js [hunk 2]
```diff
@@ -216,20 +220,27 @@ export const Line = styled.div`
...
     &::before{
       content: '';
       position: absolute;
-      border: 2px solid red;
+      border: 3px solid red;
       left: 100%;
       transform: translateY(-50%);
       border-radius: 50%;
     }
+    @media (max-width: 790px) {
```
Comment (quirozcarlos) [inline template/Pages/Home/styles.js:240]: Same here, use mobile first

**Review by quirozcarlos — CHANGES_REQUESTED** [review]

File: template/Pages/Home/styles.js (hunk 1 above)
Comment (CarlosOrdering) [reply template/Pages/Home/styles.js]: I make the modification in the last commit

File: template/Pages/Home/styles.js (hunk 2 above)
Comment (CarlosOrdering) [reply template/Pages/Home/styles.js]: I make the modification in the last commit

**Pull request merged**